jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -88,6 +88,14 @@ bool is_clinit_function(const irep_idt &function_id)
   return has_suffix(id2string(function_id), clinit_function_suffix);
 }
 
+/// Check if function_id is a user-specified clinit
+/// \param function_id: some function identifier
+/// \return true if the passed identifier is a clinit
```

Comment:
⛏ Comments go into headers.
⛏ I think the parameter and return value type are simle enough the first line of this comment would be enough to explain what this function is doing.